rfcs/0020-release-eng-process.md

```diff
@@ -0,0 +1,111 @@
+## Summary
+[summary]: #summary
+
+Moving to a Git Flow based release process with a hardened release branch and nice release cadence.
+
+## Motivation
+
+[motivation]: #motivation
+
+We need a nice release process, and it's best to pick a standard rather than reinventing the wheel because there is nice tooling available and resources online to get help.
+
+## Detailed design
+
+[detailed-design]: #detailed-design
+
+### Real Release Branches
+
+This time our "release/beta" branch wasn't really a release branch. In the future, these will be frozen and only critical user-impacting bug fixes will be allowed in. If you do want to make one of these fixes, make a PR against release branch and you'll need to include an explanation as to why you think this PR deserves to land, and you'll need approval from a `release-arbiter`. For now, Brandon will be the sole release arbiter, with the intention of introducing a program to ramp up others and set up a rotation for primary and secondary for a given release.
+
+If the fix for some critical bug is too complicated, the offending commit, if it can be found, will be reverted. If neither of these things can happen, we'll miss our release and this should be considered *bad* and all hands on deck to fix.
+
+During the period in which the release branch is open, any bugs assigned to you should be considered very high priority.
+
+### Schedule
+
+For now, we'll plan on releasing every Monday @ 2pm, starting on 8/5
+
+#### Next week:
+
+- Monday day, move `master -> develop` and `stable -> master`
```

Comment:
I don't see other mentions of `stable` ... where does that fit with other branches?

Reply:
Stable is a temporary hack that we aliased what formerly was `release/beta`